sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/PhpVersion.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> PHP version. </summary>
+    public readonly partial struct PhpVersion : IEquatable<PhpVersion>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="PhpVersion"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public PhpVersion(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string Seven2Value = "7.2";
+        private const string Seven3Value = "7.3";
+        private const string Seven4Value = "7.4";
+
+        /// <summary> 7.2. </summary>
+        public static PhpVersion Seven2 { get; } = new PhpVersion(Seven2Value);
+        /// <summary> 7.3. </summary>
+        public static PhpVersion Seven3 { get; } = new PhpVersion(Seven3Value);
+        /// <summary> 7.4. </summary>
+        public static PhpVersion Seven4 { get; } = new PhpVersion(Seven4Value);
```

Comment:
Same suggestion as above.
```suggestion
        public static PhpVersion Php7_2 { get; } = new PhpVersion(Seven2Value);
        /// <summary> 7.3. </summary>
        public static PhpVersion Php7_3 { get; } = new PhpVersion(Seven3Value);
        /// <summary> 7.4. </summary>
        public static PhpVersion Php7_4 { get; } = new PhpVersion(Seven4Value);
```

Reply:
customized to `V7.2`, `V7.3`, `V7.4`